py/src/ai/h2o/sparkling/H2OContext.py

```diff
@@ -196,7 +196,7 @@ def getOrCreate(spark, conf=None, verbose=True, pre_create_hook=None, h2o_connec
 
     def stop(self):
```

Comment:
this is used only for explicit stop by the user, if the user does not call this stop() method in pysparkling, the shutdown hook is called and stop() method on scala side is invoked